### PR TITLE
Enable strict typing

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

**Comment by balloob** [comment]
Should we wait for #128 ?

**Comment by thecode** [comment]
> Should we wait for #128 ?

Yes we wait for #128

File: aioshelly/wsrpc.py [hunk 1]
```diff
@@ -231,7 +231,7 @@ async def call(
 
         if "result" in resp:
             _LOGGER.debug("%s(%s) -> %s", call.method, call.params, resp["result"])
-            return resp["result"]
+            return dict(resp["result"])
```
Comment (bieniu) [inline aioshelly/wsrpc.py:234]: I think we should use `cast()` instead of `dict()`.

**Review by thecode — APPROVED** [review]
LGTM

File: aioshelly/wsrpc.py (hunk 1 above)
Comment (chemelli74) [reply aioshelly/wsrpc.py]: Fixed

**Review by thecode — APPROVED** [review]

**Review by bieniu — APPROVED** [review]
LGTM 👍

**Pull request merged**